src/features/jumpNav.tsx

```diff
@@ -480,9 +488,9 @@ const toggleBlockViewType = () => {
       )[0]?.[0] as PullBlock
     )?.[":children/view-type"];
     const newViewType =
-      viewType === ":document"
+      viewType === "document"
         ? "numbered"
-        : viewType === ":numbered"
+        : viewType === "numbered"
```

Comment:
fix for `Toggle Block View`

Reply:
Unfortunately this gives a type error of `This comparison appears to be unintentional because the types '":document" | ":bullet" | ":numbered" | undefined' and '"document"' have no overlap.ts(2367)`

I'm not sure where that's coming from.

`viewType` is of type `PullBlock`
`PullBlock` has `ViewType`
`ViewType` is `export declare type ViewType = "document" | "bullet" | "numbered";`
https://github.com/dvargas92495/roamjs-components/blob/main/src/types/native.ts#LL236C33-L236C33

Reply:
I'm pretty sure at one point Roam changed what gets returned and I never know for sure which one is returned.

We could resolve this without touching ViewType by using a regex: `/^:?document$/.test(viewType)`